clientEncryptionToCPKNMigrationSamples/ClientSideKeyVaultKeyToCustomerManagedKey/src/main/java/setup/Setup.java

```diff
@@ -0,0 +1,164 @@
+package setup;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.cryptography.AsyncKeyEncryptionKey;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.security.keyvault.keys.KeyClient;
+import com.azure.security.keyvault.keys.KeyClientBuilder;
+import com.azure.security.keyvault.keys.cryptography.KeyEncryptionKeyClientBuilder;
+import com.azure.security.keyvault.keys.cryptography.models.KeyWrapAlgorithm;
+import com.azure.security.keyvault.keys.models.CreateRsaKeyOptions;
+import com.azure.security.keyvault.keys.models.KeyVaultKey;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobServiceClient;
+import com.azure.storage.blob.BlobServiceClientBuilder;
+import com.azure.storage.blob.specialized.cryptography.EncryptedBlobClient;
+import com.azure.storage.blob.specialized.cryptography.EncryptedBlobClientBuilder;
+import com.azure.storage.common.StorageSharedKeyCredential;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.time.OffsetDateTime;
+import java.util.Properties;
+
+/**
+ * Set up by creating client-side encrypted blob in a new container using key vault and creates encryption scope for use
+ * in server-side encryption later
+ */
+public class Setup {
+    /**
+     * Creates an Async key for client-side encryption
+     */
+    private static AsyncKeyEncryptionKey createAsyncKey(KeyVaultKey key, TokenCredential cred) {
+        AsyncKeyEncryptionKey akek = new KeyEncryptionKeyClientBuilder()
+                .credential(cred)
+                .buildAsyncKeyEncryptionKey(key.getId())
+                .block();
+        return akek;
+    }
+
+    /**
+     * Creating encryption scope that allows access to key from key vault
+     */
+    private static void createEncryptionScope(String keyVaultUrl, String keyName, String encryptionScope,
+                                              String storageAccount, String resourceGroup, String subscription) {
+        // Creating key client that allows access of key vault
+        KeyClient keyClient = new KeyClientBuilder()
+                .vaultUrl(keyVaultUrl)
+                .credential(new DefaultAzureCredentialBuilder().build())
+                .buildClient();
+        // Get key for decryption using keyClient
+        KeyVaultKey key = keyClient.getKey(keyName);
+        String keyId = key.getId();
+        String command = "az storage account encryption-scope create --name " + encryptionScope + " -s " +
```

Comment:
Can you add a comment here with a template of the command you are running? Just in case the user wants to do the setup themselves - it may be easier for them to copy paste that